### PR TITLE
#49 Removes necessity of permissions file from the entity module.

### PR DESCRIPTION
**Pull request by Jaesin**

### CODE REVIEW

File: src/Entity/Access/EntityPermissionProvider.php [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    // Get the referring entity definition.
+    $entity_definition = $this->entity_manager->getDefinition($entity_id);
+    $type_id = $type->id();
+    $type_params = [
+      '%entity_label' => $entity_definition->getLabel(),
+      '%type_name' => $type->label(),
+    ];
+
+    return [
+      "create $type_id $entity_id" => [
```
Comment (heddn) [inline src/Entity/Access/EntityPermissionProvider.php]: This has the order backwards. It should be entity_id, followed by type_id.

**Comment by Jaesin** [comment]
This is ready for review! Anyone?

File: src/Entity/Access/EntityBasePermissions.php [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
 use Drupal\Core\StringTranslation\StringTranslationTrait;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * Defines a class containing permission callbacks.
+ *
+ * @deprecated: This (typically proxied) class will be removed in the 8.x-2.x
+ * version of CEB in favor of using the `permission_provider` entity handler.
  */
 class EntityBasePermissions implements ContainerInjectionInterface {
```
Comment (dawehner) [inline src/Entity/Access/EntityBasePermissions.php:19]: Can we drop it then, given its just needed in 8.0.x?

File: README.md [hunk 3]
```diff
@@ -12,3 +12,10 @@ You can get started with this module by copying the tests/modules/ceb_test modul
 @todo Add Drupal console support to this module.
 
 This is currently a pre-release version but has been moderately tested with Drupal 8.0.3.
+
+
```
Comment (dawehner) [inline README.md:16]: Do we have to report an issue which is just a problem in 8.0.x?

**Comment by dawehner** [comment]
Looks pretty alright for me

**Pull request merged**